modules/nodes-containers-dev-fuse-configuring.adoc

```diff
@@ -11,56 +11,66 @@ As an alternative to the virtual filesystem, you can configure the `/dev/fuse` d
 .Procedure
 
 . Create the pod.
-+
+
 [source,terminal]
 ----
-$ oc exec -ti no-priv -- /bin/bash
+oc apply -f podman-pod.yaml
 ----
-+
-[source,terminal]
+
+[IMPORTANT]
+====
+Ensure that the pod is created and running commands before continuing this procedure. Here is an example configuration similar to what you should see.
+
+[source,yaml]
 ----
-$ cat >> Dockerfile <<EOF
-FROM registry.access.redhat.com/ubi9
-EOF
+apiVersion: v1
+kind: Pod
+Metadata:
+name: podman-pod
+Annotations:
+io.kubernetes.cri-o.Devices: "/dev/fuse"
+Spec:
+containers:
+- name: podman-container
+image: quay.io/podman/stable
+       Args:
+- sleep
+- "1000000"
+securityContext:
+runAsUser: 1000
 ----
-+
+====
+
+
+. Access the shell.
 [source,terminal]
 ----
-$ podman build .
+oc exec -ti podman-pod -- /bin/bash
 ----
 
-. Implement `/dev/fuse` by adding the `/dev/fuse` device to the `io.kubernetes.cri-o.Devices` annotation.
-+
-[source,yaml]
-----
-io.kubernetes.cri-o.Devices: "/dev/fuse"
+[NOTE]
+====
+If you receive a `Permission denied` error, you might be in the wrong working directory. If this happens, specify the correct directory.
+
+[source,terminal]
 ----
-+
-For example:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: Pod
-metadata:
-  name: podman-pod
-  annotations:
-    io.kubernetes.cri-o.Devices: "/dev/fuse"
+oc exec -ti podman-pod -- /bin/bash -c "cd /desired/directory && bash"
 ----
+====
 
-. Configure the `/dev/fuse` device in your pod specifications.
-+
-[source,yaml]
+. Create a Dockerfile and build the image.
+
+[source,terminal]
 ----
-spec:
-  containers:
-  - name: podman-container
-    image: quay.io/podman/stable
-    args:
-    - sleep
-    - "1000000"
-    securityContext:
-      runAsUser: 1000
+cat >> Dockerfile <<EOF
+FROM registry.access.redhat.com/ubi9
+EOF
+podman build .
 ----
 
+. Ensure the `/dev/fuse` device is available and properly configured:
 
+[source,terminal]
+----
+ls /dev/fuse
+----
```

Comment:
Do we know what the output of that command would be if the `/dev/fuse` device is available and properly configured?